### ISSUE TITLE
Chombo datasets unpack inconsistently

### ISSUE THREAD

**Issue by munkm** [yt-project/website#73]
I've been going through and downloading the frontend data from our webpage. When downloading the five datasets for the chombo frontend I get inconsistent ways that they're unpacked. Three of them unpack into folders similarly named to their .tar.gz folder. Those are:
- IsothermalSphere.tar.gz --> `IsothermalSphere/`
- KelvinHelmholtz.tar.gz --> `KelvinHemholtz/`
-ZeldovichPancake.tar.gz --> `ZeldovichPancake/`

Two of the zipped files unzip the data file into the current directory. Those are:
TurbBoxLowRes.tar.gz --> data.0005.3d.hdf5
GaussianCloud.tar.gz --> data.0077.3d.hdf5 

I think we probably should have consistent unzipping for each frontend? Looking at the frontend it also seems that the latter two files are expected to be located in `"GaussianCloud/data.0077.3d.hdf5"` and `"TurbBoxLowRes/data.0005.3d.hdf5"`, respectively.

**Comment by ngoldbaum** [comment]
This was probably just an oversight when those datasets were originally added. I've been trying to make sure the datasets all unpack into folders with useful names rather than as files but there are some that don't do that. I'd personally say it's fine to just fix this in-place on the webserver -- I'd be surprised if anyone was depending on the name of the directory that these files get unpacked into.

**Comment by munkm** [comment]
Yeah, I don't think it's a huge deal. I just wanted to make sure the issue was documented somewhere.
